nextcord/endpoint/__init__.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
+
+import logging
+
+from aiohttp import web
+from aiohttp.typedefs import Handler
+from aiohttp.web import StreamResponse
+
+from ..enums import OAuth2Scopes
+from .base import BaseEndpoint
+from .oauth import OAuth2Endpoint, oauth_url_request_generator
+from .role_conn_verify import RoleConnVerifyEndpoint, role_conn_metadata_generator
+
+_log = logging.getLogger(__name__)
+
+
+__all__ = (
+    "oauth_url_request_generator",
+    "OAuth2Endpoint",
+    "role_conn_metadata_generator",
+    "RoleConnVerifyEndpoint",
+    "UnifiedEndpoint",
+)
+
+
+class UnifiedEndpoint(BaseEndpoint):
+    def __init__(
+        self,
+        oauth2: bool = True,
+        role_conn: bool = True,
```

Comment:
`role_conn` as a variable without documention doesn't implie a lot of what it does to me personally but if this is internally facing it should be fine